tests/test_frame.py

```diff
@@ -1087,6 +1087,18 @@ def test_types_pivot_table() -> None:
         pd.DataFrame,
     )
 
+    def test_pivot_table_sort():
+        df = pd.DataFrame(
+            {"a": [1, 2],
+             "b": [3, 4],
+             "c": [5, 6],
+             "d": [7, 8]}
+        )
+
+
+        check(assert_type(df.pivot_table(values="a", index="b", columns="c"
+                                         , sort=True), pd.DataFrame), pd.DataFrame)
+
```

Comment:
CI is failing. 

dedent this - should be at same level as function above.

Also, make sure to do the pre-commit steps.  This needs to have `black` applied.